django/core/management/commands/inspectdb.py

```diff
@@ -111,6 +111,9 @@ def table2model(table_name):
                             cursor, table_name
                         )
                     )
+                    composite_key = connection.introspection.get_composite_key(
+                        cursor, table_name
+                    )
```

Comment:
So use one method instead of two different methods right??

Reply:
> So use one method instead of two different methods right??

Yes, so use `get_composite_key()` (or `get_primary_key_columns()`), e.g.
```python
primary_key_columns = connection.introspection.get_primary_key_columns(cursor, table_name)
primary_key_column = primary_key_columns[0] if primary_key_columns else None
...
for row in table_description:
    ...
    if column_name == primary_key_column:
        extra_params["primary_key"] = True
        if len(primary_key_columns) > 1:
            comment_notes.append(...)
```